### PR TITLE
Changes for v9.0-rc1 prerelease

### PR DESCRIPTION
**Pull request by rvem**
## Description
Problem: v9.0-rc1 arrived, we should create a pre-release for it.

Solution: Bump Tezos sources.

## Related issue(s)

Resolves None

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

- [x] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: docker/build/static_libs.patch [hunk 1]
```diff
@@ -96,96 +96,6 @@ index 7f2ec8a63..51ffdcc8b 100644
   (modules Main_native))
  
  (executable
-diff --git a/src/proto_006_PsCARTHA/bin_accuser/dune b/src/proto_006_PsCARTHA/bin_accuser/dune
```
Comment (balsoft) [inline docker/build/static_libs.patch:99]: Reviewing diffs of diffs :kaef:

**Review by balsoft — APPROVED** [review]
Superficially looks ok.

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -69,50 +69,50 @@ steps:
...
+#    - eval "$SET_VERSION"
+#    # Building all binary packages will take significant amount of time, so we build only one
+#    # in order to ensure package generation sanity
+#    - ./docker/docker-tezos-packages.sh --os fedora --type binary --package tezos-baker-008-PtEdo2Zk
+#    - rm -rf out
+#    # It takes much time to build binary package, so we do it only on master
+#    branches: "master"
+#    timeout_in_minutes: 90
+#    agents:
+#      queue: "docker"
```
Comment (balsoft) [inline .buildkite/pipeline.yml:115]: Don't forget about this :P

**Review by balsoft — APPROVED** [review]

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (rvem) [reply .buildkite/pipeline.yml:115]: This is going to be uncommented only once Tezos will release v9.0 opam packages

**Pull request merged**